File: Função Calc/Calc.php
<?php
echo("Bem vindo a calculadora do PP\n");
echo("digite um número: ");
$num1 = intval(fgets(STDIN));  //Lê o valor de um int
echo("Digite outro número: ");
$num2 = intval(fgets(STDIN));  //Lê outro valor de um int
function Soma($x,$y){
    return ($x + $y);
}
function  Subtracao($x, $y) {
    return ($x - $y);
}
function Multiplicacao($x, $y) {
    return ($x * $y);
}
function Divisao($x, $y) {
    if($y == 0){
        echo "Erro! Não é possível dividir por zero.\n";
    }else{
        return ($x / $y);
    }   
}
echo("Escolha:\n 1-Soma 2-Subtração 3-Multiplicação 4-Divisão\n");
$escolha = intval(fgets(STDIN));
if ($escolha == 1){
    echo Soma( $num1 , $num2 );
}elseif ($escolha == 2){
    echo Subtracao( $num1 , $num2 );
}elseif ($escolha == 3){
    echo Multiplicacao( $num1 , $num2 );
}elseif ($escolha ==4){
    echo Divisao( $num1 , $num2 );
}else {
    echo("Não achamos o operador");
}
echo("Isso foi a calculadora Paulo Petrola!");
?>

File: Função Calc/Calc.java
class Calc {
    public static void main(String[] args){
        int Escolha = (int)(Math.random()*4+1);
        System.out.println("Escolha: \n 1 - Soma \n 2 - Subtração \n 3 - Multiplicação \n 4 - Divisão ");
        System.out.println("Escolheu:\n" + Escolha);
        int x = (int) (Math.random()*10+1);
        System.out.println("Primeiro Número:\n" + x);
        int y = (int) (Math.random()*10+1);
        System.out.println("Segundo Número:\n" + y);

        if(Escolha == 1){
        System.out.println("Resultado\n" + (x+y));
        }else if(Escolha == 2){
        System.out.println("Resultado\n"+ (x-y));
        }else if (Escolha == 3){
        System.out.println("Resultado\n"+ (x*y));
        }else if (Escolha == 4){
         System.out.println("Resultado\n"+ (x/y));
        }else{
        System.out.println("opcão inválida");
        }
    
}
    }